### PR TITLE
Feature/card css

### PR DESCRIPTION
**Pull request by domingobishop**

### CODE REVIEW

File: css/sass/_cards.scss [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+      display: block;
+      width: 100%;
+    }
+    h3 {
+      margin: .5em 0;
+    }
+    p {
+      color: #444;
+    }
+    a:hover {
```
[the remark this answers is not in the document]
Comment (domingobishop) [reply css/sass/_cards.scss]: Done

File: css/sass/_cards.scss [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+  p {
+    margin-bottom: .5em;
+  }
+  @media (min-width: 768px) {
+    display: table;
+    min-height: 240px;
+    a {
+      display: table-row;
+    }
+    .entry-image {
```
[the remark this answers is not in the document]
Comment (domingobishop) [reply css/sass/_cards.scss]: Done

File: css/sass/_portal-landing.scss [hunk 3]
```diff
@@ -0,0 +1,51 @@
+.portal-landing {
+  .banner {
```
[the remark this answers is not in the document]
Comment (domingobishop) [reply css/sass/_portal-landing.scss]: Done

File: js/tna-global.js [hunk 4]
```diff
@@ -26,3 +26,17 @@
     }
 }(jQuery));
 
+$(function() {
+    $('a[class="anchor-link"]').click(function() {
```
[the remark this answers is not in the document]
Comment (domingobishop) [reply js/tna-global.js]: Done

**Comment by punalsc** [comment]
All looks good.

**Pull request merged**